src/pages/test-report/cat-cpc/components/question-del-ex-radio-card/question-del-ex-radio-card.html

```diff
@@ -0,0 +1,91 @@
+<ion-card>
+  <question-title [question]="question"></question-title>
+
+  <ion-card-content>
+
+    <ion-grid>
```

Comment:
Have you considered implementing this as data driven rows?

What I mean by that is to have an array that contains [0, 5, 10, 15, 20] which you can map through and render one column per element.

Reply:
Good shout - refactored to repeat row from data .  not sure why i didnt in the first place, i did exactly that when calling the component.

Reply:
Nice one! Looks pretty neat 

While you're removing the `console.log` below, can you please create the array of [0, 5, 10, 15, 20] as a component variable? So that it's much more visible that it's actually data driven and potentially that array can come from anywhere